assets/csidriveroperators/kubevirt/07_deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubevirt-csi-driver-operator
+  namespace: openshift-cluster-csi-drivers
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: kubevirt-csi-driver-operator
+  template:
+    metadata:
+      labels:
+        name: kubevirt-csi-driver-operator
+    spec:
+      serviceAccountName: kubevirt-csi-driver-operator
+      priorityClassName: system-cluster-critical
+      containers:
+        - name: kubevirt-csi-driver-operator
+          imagePullPolicy: Always
```

Comment:
should be `IfNotPresent`

Reply:
I don't mind changing it. However, we used Always in CSI driver and operator.

Reply:
done